src/test/regress/sql/subselect.sql

```diff
@@ -1069,3 +1069,23 @@ drop table tl1;
 drop table tl2;
 drop table tl3;
 drop table tl4;
+
+--Test case for subquery, which returns more than one rows
+-- start_ignore
+DROP TABLE IF EXISTS table1, table2;
+-- end_ignore
+CREATE TABLE table1 (a INT, b INT) DISTRIBUTED BY (a);
+CREATE TABLE table2 (a INT, b INT) DISTRIBUTED BY (a);
+
+INSERT INTO table1 VALUES (1, 0);
+INSERT INTO table1 VALUES (1, 0);
+INSERT INTO table2 VALUES (0, 10);
+INSERT INTO table2 VALUES (0, 10);
```

Comment:
I suggest to create and fill the tables using one query per table
```suggestion
create table table1 as 
    select * from (values (1, 0), (1, 0)) v(a, b) distributed by (a);
create table table2 as
    select * from (values (0, 10), (0, 10)) v(a, b) distributed by (a);
```